tests/NovuTest.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace Novu\SDK\Tests;
+
+use Novu\SDK\Novu;
+use Novu\SDK\Resources\Workflow;
+use PHPUnit\Framework\TestCase;
+
+class NovuTest extends Testcase
+{
+    /**
+     * @var \Novu\SDK\Novu
+     */
+    private $novu;
+
+    protected function setUp(): void
+    {
+                $this->novu = new Novu([getenv('NOVU_API_KEY')]);
+    }
+
+    public function testCreateWorkflow()
+    {
+        $workflow = $this->novu->createWorkflow(
+            [
+                'name'                => 'Onboarding Workflow',
+                'notificationGroupId' => '5f5f5f5f5f5f5f5f5f5f5f5f',
+                'steps'               => [
+                    [
+                        'active'           => true,
+                        'shouldStopOnFail' => false,
+                        'uuid'             => '78ab8c72-46de-49e4-8464-257085960f9e',
+                        'name'             => 'Chat',
+                        'filters'          => [
+                            [
+                                'value'    => 'AND',
+                                'children' => [
+                                    [
+                                        'field'    => '{{chatContent}}',
+                                        'value'    => 'flag',
+                                        'operator' => 'NOT_IN',
+                                        'on'       => 'PAYLOAD',
+                                    ],
+                                ],
+                            ],
+                        ],
+                        'template' => [
+                            'type'      => 'chat',
+                            'active'    => true,
+                            'subject'   => '',
+                            'variables' => [
+                                [
+                                    'name'     => 'chatContent',
+                                    'type'     => 'STRING',
+                                    'required' => true,
+                                ],
+                            ],
+                            'content'     => '{{chatContent}}',
+                            'contentType' => 'editor',
+                        ],
+                    ],
+                ],
+                'description' => 'Onboarding workflow to trigger after user sign up',
+//                'active'      => true,
```

Comment:
When passing these values, it does not detect them as boolean and throws an 400 error. Do you have any idea why??

Reply:
Please can you shed more light on this. Please provide more clarity on what you mean.

Reply:
I mean when I set one of the  (`active`,`draft`,`critical`) it throws  a 400 error and says that it must be a boolean value (which I am setting that ofc ;)

Reply:
Did you test this on the docs api reference. What was your response there? https://docs.novu.co/api-reference/workflows/create-workflow